homeassistant/components/ozw/discovery.py

```diff
@@ -250,6 +250,16 @@
                 const.DISC_INDEX: ValueIndex.SWITCH_COLOR_CHANNELS,
                 const.DISC_OPTIONAL: True,
             },
+            "min_kelvin": {
+                const.DISC_COMMAND_CLASS: (CommandClass.CONFIGURATION,),
+                const.DISC_INDEX: ValueIndex.SENSOR_MULTILEVEL_OZONE,  # PR for upstream to add LIGHT_COLOR_KELVIN_MIN
```

Comment:
Let's use the integers directly until the library is updated.

Reply:
We tossed around the idea to define a const for it in const.py as well.

Reply:
We want to define these in the library. But until we have that it's confusing and a bit dangerous to use the value index of another command class.

Reply:
We have more places where we need to replace integers with constants from the library. So it's fine for now.